polaris-react/src/components/TextField/tests/TextField.test.tsx

```diff
@@ -1399,6 +1399,351 @@ describe('<TextField />', () => {
         });
       });
     });
+
+    describe('integer', () => {
```

Comment:
Most of these tests are copied over from the number type. I feel like perhaps we could simplify by merging common tests between the two into a single "numeric type" test